src/renderer/i18n/en.ts

```
@@ -43,7 +43,28 @@
         contents: "Slide Contents\nMarkdown bullets\n- one\n- two",
       },
       markdown: {
-        contents: "Markdown Contents\n# Title\nWrite your content here...\n- Item 1\n- Item 2\n- Item 3",
+        contents: "Markdown Contents\n# Title\nWrite your content here.\n- Item 1\n- Item 2\n- Item 3",
+      },
+      htmlPrompt: {
+        contents: "Enter HTML prompt to generate custom slide content.",
+      },
+      chart: {
+        contents: "Enter chart data in JSON format\n" + '{\n}',
```

ESLint

[error] Replace `'{\n}'` with `"{\n}"`

(prettier/prettier)